tests/trace/test_dictifiable.py

```diff
@@ -0,0 +1,45 @@
+import weave
+
+
+def test_dictifiable(client):
+    class NonDictifiable:
+        attr: int
+
+        def __init__(self, attr: int):
+            self.attr = attr
+
+    class Dictifiable:
+        attr: int
+
+        def __init__(self, attr: int):
+            self.attr = attr
+
+        def to_dict(self):
+            return {"attr": self.attr}
+
+    @weave.op
+    def func(d: Dictifiable, nd: NonDictifiable) -> Dictifiable:
```

Comment:
This return type seems wrong, it's a dict, not a Dictifiable?

Reply:
fixed - i originally had it as dictifiable, but wanted to make sure both worked.

Reply:
I see we exclude the `tests` dir from `mypy`